package.json

```diff
@@ -49,7 +49,10 @@
   },
   "homepage": "https://github.com/SU-SWS/decanter-react#readme",
   "dependencies": {
+    "@emotion/react": "^11.4.1",
```

Comment:
So here's where things start getting interesting (read convoluted); Apparently mui@v5 is now using emotion 👩🏼‍🎤  (which is a fantastic css-in-js solution if that's your thing) so we're adding that to our bundle now, too. Something to further discuss in dev talk...